src/main/java/rueppellii/backend2/tribes/user/ApplicationUserService.java

```diff
@@ -76,7 +67,13 @@ private String kingdomNameValidation(String kingdomName, String userName) {
         return kingdomName;
     }
 
-    public ResponseEntity<?> authenticateApplicationUser(LoginForm loginForm)
+    private Kingdom createNewKingdom(SignUpForm signUpForm) {
```

Comment:
"createNewKingdom" is not enough accurate, because you set kingdom's name as well

Reply:
fixed and pushed